src/datamodel/game/player.rs

```diff
@@ -0,0 +1,6 @@
+use crate::datamodel::character::Character;
+
+pub struct Player {
```

Comment:
Shouldn't `Player` contain the character's ability too?

Reply:
It has a `character` field which contains the character associated with the player.

Reply:
Ok, so, we have to add that to the `character` then. #10

Reply:
I commented in the issue #10

Reply:
I'll resolve here to move on then